api/admin_api.yaml

```diff
@@ -0,0 +1,298 @@
+                  serviceName: service
+                  id: 9d39a48a64370b1b326dfb70307008-0000000000000000
+                  version: 0
+                  authorizedCIDRs: []
+                  kind: IRetrievedService
+      description: An endpoint to debug POST requests to the API backend.
+      summary: Debug POST
+  "/service":
+    post:
+      responses:
+        '200':
+          description: Service created.
+          schema:
+            "$ref": "#/definitions/AdmService"
+          examples:
+            application/json:
+              id: 2b3e728c1a5d1efa035c-0000000000000001
+              authorizedRecipients:
+              - XXXYYY79A95Y000X
+              departmentName: dept
+              organizationName: org
+              serviceId: 2b3e728c1a5d1efa035c
+              serviceName: service
+              version: 1
+              authorizedCIDRs: []
+      summary: Create Service
+      description: Create a new Service with the attributes provided in the requst
+        payload.
+      operationId: createService
+      parameters:
+      - in: body
+        name: body
+        schema:
+          "$ref": "#/definitions/AdmService"
+        description: The Service payload.
+        x-examples:
+          application/json:
+            authorizedRecipients:
+            - XXXYYY79A95Y000X
+            departmentName: dept
+            organizationName: org
+            serviceId: 2b3e728c1a5d1efa035c
+            serviceName: service
+            authorizedCIDRs: []
+definitions:
+  AdmService:
+    title: AdmService
+    description: A Service tied to an user's subscription.
+    type: object
+    properties:
+      serviceId:
+        type: string
+        description: The ID of the Service. Equals the subscriptionId of a registered
+          API user.
+      serviceName:
+        type: string
+        description: The name of the service. Will be added to the content of sent
+          messages.
+      organizationName:
+        type: string
+        description: The organizazione that runs the service. Will be added to the
+          content of sent messages to identify the sender.
+      departmentName:
+        type: string
+        description: The departmenet inside the organization that runs the service.
+          Will be added to the content of sent messages.
+      authorizedCIDRs:
+        type: array
+        items:
+          "$ref": "#/definitions/AdmCidr"
+      authorizedRecipients:
+        type: array
+        items:
+          "$ref": "#/definitions/AdmFiscalCode"
+      version:
+        type: integer
+      id:
+        type: string
+    required:
+    - serviceId
+    - serviceName
+    - organizationName
+    - departmentName
+    - authorizedCIDRs
+    - authorizedRecipients
+  AdmCidr:
+    type: string
+    title: AdmCidr
+    description: Describes a single IP or a range of IPs.
+    pattern: "([0-9]{1,3}\\.){3}[0-9]{1,3}(\\/([0-9]|[1-2][0-9]|3[0-2]))?"
+  AdmFiscalCode:
```

Comment:
why we need an `AdmFiscalCode`? we already have a definition for `FiscalCode`

Reply:
I opted to keep the API spec "self contained", othwerwise we got:

    $ref : "public_api_v1.yaml#/definitions/FiscalCode"

which introduces some non-trivial problems when generating the models from the specs
(even calling Swagger.bundle() instead of Swagger.parse() does not solve the issue).

That's the reason why I opted for using a prefix/namespace ("Adm") to the models definitions, in this way they does not conflict with the other ones.

I suggest to keep thie behavoir and open an issue if we really need to share definitions between specs files (which is a broader problem than the aim of this PR)

Reply:
I've updated the specs extracting the definitions in another YAML file. The drawback is that the whole definition file is bundled into the two specs (even the unused definitions). Aside the bigger swagger.json size it does not impact any other aspect.